matrix-vis.js

```diff
@@ -105,11 +105,24 @@ function initGraph() {
     network = new vis.Network(container, data, options);
     // add event listeners
     network.on('select', function(params) {
-        var text = "";
+        var Type = " ";
+        var Sender = " ";
```

Comment:
Variable names in Javascript should always start with a lower-case letter. Class names with an upper-case letter. Constants in ALL_CAPS. Please lower case `Type` and `Sender`.